invenio_rdm_records/fixtures/tasks.py

```diff
@@ -7,20 +7,21 @@
 # it under the terms of the MIT License; see LICENSE file for more details.
 
 """Celery tasks for fixtures."""
-import arrow
 from celery import shared_task
-from flask import current_app
 from invenio_access.permissions import system_identity
 from invenio_vocabularies.proxies import current_service as vocabulary_service
 
 from ..proxies import current_rdm_records
-from ..services.errors import EmbargoNotLiftedError
 
 
 @shared_task
-def create_vocabulary_record(data):
+def create_vocabulary_record(service_str, data):
     """Create a vocabulary record."""
-    vocabulary_service.create(system_identity, data)
+    if service_str == "vocabulary_service":
+        service = vocabulary_service
+    else:
+        service = getattr(current_rdm_records, service_str)
```

Comment:
I do agree with favouring the import of a specific service when available instead of using the registry... But here, why not use the registry? It exists and is a bit nicer than `getattr` IMO

Reply:
We discussed this during the standup (at least this is what I took out of it): we don't need to use the registry and we let the registry be its thing in its own corner.

The registry is a "workaround" for access to services by id in a module that cannot simply import the service because of circular dependency. The concrete case why it was created is this file: https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/tasks.py (read the commit comments there too). Basically, a generic task was placed in records-resources that would use a service in packages depending on records-resources (it can't just import them). 

Other packages (invenio-rdm-records) that have direct import access don't need to use the registry.

Reply:
I do recall that, that's why I agree on favouring importing specific services... However, since the workaround already exists I  think it's better to use it than `getattr`... But I guess that's a design choice.